FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/Views/AuthPickerView.swift

```diff
@@ -0,0 +1,51 @@
+import SwiftUI
+
+@MainActor
+public struct AuthPickerView<Content: View> {
+  @Environment(AuthService.self) private var authService
+  let providerButtons: () -> Content
+
+  public init(@ViewBuilder providerButtons: @escaping () -> Content) {
+    self.providerButtons = providerButtons
+  }
+
+  private func switchFlow() {
+    authService.authenticationFlow = authService
+      .authenticationFlow == .login ? .signUp : .login
+  }
+}
+
+extension AuthPickerView: View {
+  public var body: some View {
+    VStack {
+      if authService.authenticationState == .authenticated {
+        SignedInView()
+      } else if authService.authView == .passwordRecovery {
+        PasswordRecoveryView()
+      } else if authService.authView == .emailLink {
+        EmailLinkView()
+      } else {
+        Text(authService.authenticationFlow == .login ? "Login" : "Sign up")
+        VStack { Divider() }
+        EmailAuthView()
```

Comment:
So this means that we'll always have Email/Password be the most prominent provider. I would advocate to allow users to instantiate EmailAuthView themselves via the providerButtons closure.

1) I think we should no longer promote Email/Password as the go-to auth provider. Passwords are inherently insecure, and we should promote more secure providers instead.
2) This gives developers more flexibility (e.g. put SiwA / SiwG first, and Email last)